Modules/@babylonjs/react-native/shared/BabylonNative.h

```diff
@@ -10,6 +10,12 @@ namespace Babylon
     void Deinitialize();
     void UpdateView(void* windowPtr, size_t width, size_t height, void* windowTypePtr = nullptr);
     void RenderView();
-    void SetPointerButtonState(uint32_t pointerId, uint32_t buttonId, bool isDown, uint32_t x, uint32_t y);
-    void SetPointerPosition(uint32_t pointerId, uint32_t x, uint32_t y);
+    void SetMouseButtonState(uint32_t buttonId, bool isDown, uint32_t x, uint32_t y);
+    void SetMousePosition(uint32_t x, uint32_t y);
+    void SetTouchButtonState(uint32_t pointerId, bool isDown, uint32_t x, uint32_t y);
+    void SetTouchPosition(uint32_t pointerId, uint32_t x, uint32_t y);
+
+    extern const uint32_t LEFT_MOUSE_BUTTON_ID;
+    extern const uint32_t MIDDLE_MOUSE_BUTTON_ID;
+    extern const uint32_t RIGHT_MOUSE_BUTTON_ID;
```

Comment:
Just make these `constexpr`?

Reply:
if we want these values to map to the values in NativeInput.h they need to be defined in a cpp file. Otherwise we have to reference the NativeInput header forcing all BabylonNative.h consumers to start having to reference NativeInput include folders.

if we just want to re-hardcode these values here we could use a constexpr, but that doesn't ensure we don't get out of sync with native input definitions

Reply:
Ah right, I think you said that elsewhere and I don't think it quite connected. I understand now and I think what you have makes sense.